### PR TITLE
[Polish UI] Update of the download page

### PR DESCRIPTION
**Pull request by Patriot99**
Related to PR [#1397](https://github.com/Nexus-Mods/NexusMods.App/pull/1397)

### CODE REVIEW

File: src/NexusMods.App.UI/Resources/Language.pl.resx [hunk 1]
```diff
@@ -456,6 +456,18 @@
...
+		<value>DZIAŁANIA</value>
+	</data>
+    <data name="DownloadStatusDesignViewModel_FormatStatus_Cancelled" xml:space="preserve">
+        <value>Anulowane</value>
+    </data>
+    <data name="DownloadInProgressView__Clear" xml:space="preserve">
+        <value>Wyczyść</value>
+    </data>
+    <data name="DownloadInProgressView__Clear_completed" xml:space="preserve">
+        <value>Wyczyść ukończone</value>
```
Comment (Sewer56) [inline src/NexusMods.App.UI/Resources/Language.pl.resx:468]: It this how software usually phrases it? The translation is quite literal.

I was wondering if `wyczyszone` might not be more correct here.

Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:468]: Clear - Wyczyść
Cleared - Wyczyszczone

I haven't checked the UI yet, but I suspect it is a button/option to clear completed downloads. Something similar to the Clear All option in the download history in the Chrome browser. 

Wyczyść is the future tense.
Wyczyszcozne is the past tense.

Comment (Sewer56) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:468]: I was thinking of `Clear Completed` as 'we just cleared this' hence using past tense.
But as my Polish is not very advanced, I wanted to doublecheck.

Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:468]: Wyczyść ukończone is the literal translation of Clear completed. It is user-friendly and easy to understand. The translation depends on the context and what the option does in the UI.

Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:468]: I don't have the option to build the application now. All I need is a screenshot (it doesn't matter what language you have the app in) to confirm that the translation is correct, if you want to be 100% sure.

Comment (Sewer56) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:468]: image

Nevermind, it is 'clear completed'. Not a response to a click. just the static text.
So your translation is ok. My bad for not checking the UI before commenting.

Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:468]: Static or not, in either case the translation is correct. Thanks for the screenshot.

**Review by Sewer56 — APPROVED** [review]

**Pull request merged**